js/assessments/sentenceBeginningsAssessment.js

```diff
@@ -0,0 +1,60 @@
+var AssessmentResult = require( "../values/AssessmentResult.js" );
+var partition = require ( "lodash/partition" );
+var sortBy = require ( "lodash/sortBy" );
+
+/**
+ * Counts the number too often used sentence beginnings and determines the lowest count within that group.
+ * @param {array} sentenceBeginnings The array containing the objects containing the beginning words and counts.
+ * @returns {object} The object containing the total number of too often used beginnings and the lowest count within those.
+ */
+var calculateSentenceBeginningsResult = function( sentenceBeginnings ) {
```

Comment:
I'd rename this to something along the lines of `calculateTotalSentenceBeginnings` and reserve the `calculateSentenceBeginningsResult` name for the method that calculates the actual assessment result, which is now being done on lines 36 - 51.

See `metaDescriptionKeywordAssessment.js` for an example of how these results were being created, even if no AssessmentResult can be calculated.